src/main/resources/org/jenkinsci/test/acceptance/docker/fixtures/SshdContainer/Dockerfile

```diff
@@ -24,4 +24,6 @@ RUN echo "KexAlgorithms curve25519-sha256@libssh.org,ecdh-sha2-nistp256,ecdh-sha
 RUN echo "MACs hmac-sha2-512-etm@openssh.com,hmac-sha2-256-etm@openssh.com,hmac-ripemd160-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-512,hmac-sha2-256,hmac-ripemd160,umac-128@openssh.com,hmac-md5,hmac-sha1,hmac-sha1-96,hmac-md5-96" >> /etc/ssh/sshd_config
 
 # run SSHD in the foreground with error messages to stderr
-ENTRYPOINT ["/usr/sbin/sshd", "-D", "-e"]
+ENTRYPOINT ["/usr/sbin/sshd", "-D", "-e", "-ddd"]
+
+RUN getent passwd
```

Comment:
What is this for?

Reply:
Chasing "No user exists for uid 1000" from ci.jenkins.io  I can not reproduce locally.